sire/intro.md

```diff
@@ -122,25 +120,9 @@ renamedInput
 
 ## Natural numbers, byte-arrays and strings
 
-The big takeaway from this section has to do with natural numbers, but we have to get a thing or two out of the way first about how the REPL deals with strings.
-
-The REPL renders strings in single curly braces, `{like this example}`. You can also enter a string this way, as in `= msg {hello world}` in order to bind the name `msg` to the string `hello world`.
-
-For a string without spaces, the REPL will opt to render it with a leading `%` as in `%houseboat`. Again, you can enter strings this way as well: `= color %blue` (which binds the string `blue` to the name `color`). You may sometimes see this style referred to as an "atom" in some source files.
-
-It's worth mentioning that you _can enter_ a string with double-quotes as you may expect, but the REPL never prints them that way:
-
-```
-= msg "hello world"
-msg
-;; prints:
-{hello world}
-```
-It is for this reason we suggest you get used to the `{}` and `%` style.
-
 ### Nat - natural number
 
-All values in Pallas are stored in memory as natural numbers. Throughout the system, these are referred to as "nats". They're sometimes also called "nouns".  
+All values in Pallas are stored in memory as natural numbers. Throughout the system, these are referred to as "nats".
```

Comment:
I know i should've mentioned this before but all values are not stored as nats.

```suggestion
All values in Pallas are trees, whose leaves are natural numbers. Throughout the system, these are referred to as "nats".
```

Reply:
crucial detail!